
math.h is needed for all aritectures without sin,cos,exp,pow buildins.

  - Rene Rebe <rene@rocklinux.org>

--- ./widget-engine/baghira.cpp.orig	2004-04-22 20:16:48.000000000 +0200
+++ ./widget-engine/baghira.cpp	2004-04-22 20:17:16.000000000 +0200
@@ -13,7 +13,7 @@
 
 // this seems to be important for some versions of gcc
 // so uncomment the following line, if you get error messages about "unkonwn function exp" or "pow"
-//#include <math.h>
+#include <math.h>
 
 #define _SCALE_    65536
 #define _NORMALIZE_ >>16
